core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -149,12 +149,12 @@
                     </div>
                     <div (keydown)="onMenuKeypress($event, 'learnMenu', {enter: ACTION_CLOSE})"
                          class="nav-item-bottom-link">
-                      <span class="underline">
+                      <a class="underline nav-item-bottom-link" tabindex="0" href="/learn/math">
```

Comment:
@SanjaySajuJacob I have added href="/learn/math" so that when the user press enters on see all lessons it will redirect it to the correct page. i think this is what you were asking from me to do